src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -2979,6 +2979,11 @@ public long[] getPerLevelSizeBytes()
         return compactionStrategyManager.getPerLevelSizeBytes();
     }
 
+    public int[] getSSTableCountPerTWCSBucket()
```

Comment:
nit: add `@Override`

Reply:
Will go ahead and fix the other MBean methods that are missing it as well. 👍